marklogic-data-hub/src/main/java/com/marklogic/hub/flow/FlowRunner.java

```diff
@@ -23,18 +23,81 @@
 
 public interface FlowRunner {
 
+    /**
+     * Sets the flow to be used with the flow runner
+     * @param flow the flow object to be used
+     * @return the flow runner object
+     */
     FlowRunner withFlow(Flow flow);
+
+    /**
+     * Sets the batch size for the flow runner
+     * @param batchSize - the size of the batch in integer form
+     * @return the flow runner object
+     */
     FlowRunner withBatchSize(int batchSize);
+
+    /**
+     * Sets the thread count for the flowrunner
+     * @param threadCount - the number of threads for the flow runner to use
+     * @return the flow runner object
+     */
     FlowRunner withThreadCount(int threadCount);
+
+    /**
+     * Sets the source client on the flow runner
+     * @param sourceClient
+     * @return  the flow runner object
+     */
     FlowRunner withSourceClient(DatabaseClient sourceClient);
+
+    /**
+     * Sets which database the flow will output to
+     * @param destinationDatabase
+     * @return the flow runner object
+     */
     FlowRunner withDestinationDatabase(String destinationDatabase);
+
+    /**
+     * Sets the options for the flowRunner
```

Comment:
Sets the options to be passed into the xqy or sjs flow in the $options or options variables of main.